src/layouts/BaseLayout.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {FC} from 'react';
+import React, {FC, useEffect} from 'react';
```

Comment:
do we need useEffect or is it unused?
```suggestion
import React, {FC} from 'react';
```

Reply:
Oh no, I just tried something out and forgot to dismiss

Reply:
:)